Examples/Swift/MapSnapshotterExample.swift

```diff
@@ -0,0 +1,51 @@
+import Mapbox
+
+@objc(MapSnapshotterExample_Swift)
+
+class MapSnapshotterExample: UIViewController, MGLMapViewDelegate {
+
+    var mapView : MGLMapView!
+    var button : UIButton!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        mapView = MGLMapView(frame: view.bounds, styleURL: MGLStyle.satelliteStreetsStyleURL())
+        mapView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+        
+        // Center map on the Giza Pyramid Complex in Egypt.
+        let center = CLLocationCoordinate2D(latitude: 29.9773, longitude: 31.1325)
+        mapView.setCenter(center, zoomLevel: 14, animated: false)
+        view.addSubview(mapView)
+
+        // Create a button to take a map snapshot.
+        button = UIButton(frame: CGRect(x: view.bounds.width / 2 - 15, y: view.bounds.height - 80, width: 30, height: 30))
```

Comment:
The current size of the button reminds me a bit of a map marker — if we made it larger, it might be more obvious that it’s a button.